tests/Dockerfile

```diff
@@ -52,6 +52,23 @@ RUN dnf -y install \
   && dnf clean all
 RUN echo "%_dbpath $(rpm --eval '%_dbpath')" > /root/.rpmmacros
 
+# Workaround for pkgconf(1)'s unlisted dependency on rpm.
+# This is needed for cmake to work without an rpm installation.
+RUN ln -sf $(rpm --eval '%{_target_platform}%{?_gnu}')-pkg-config \
```

Comment:
This is pretty sick really... (not objecting to what doing it, just grumbling as a reminder to myself to file a bug on Fedora's pkgconf)

Reply:
Yup, the funny thing is, I've also made myself a note back then to file a bug for this and of course never got around to it :smile: